doc/Configuration.md

```diff
@@ -54,26 +54,16 @@ Here you specify what you want to build. See example configs for sensible
 lists. The order of components is important - it should reflect build
 dependencies, otherwise build would fail.
 
-### CC_`component`
-> Default: Value set in Makefile of the component
+### ENV_`component`
+> Default: None
 
-Custom component specific compiler
+Specify environment variables for component build
 
-Example:
-CC_gui-daemon=clang
-
-### CFLAGS_`component`
-> Default: Value set in Makefile of the component
-
-Custom component specific compiler flags
-
-Example:
-CFLAGS_gui-daemon=-fstack-protector-strong
-
-### LDFLAGS_`component`
-> Default: Value set in Makefile of the component
+For example:
+ENV_gui-daemon = "\
```

Comment:
Also, I hope it is still possible to specify multiple flags, like "CFLAGS=-Wall -Werror", i.e. individual values with spaces, right? Probably some escaping will be needed (this example should show how to do it).